interfaces/Portal/src/app/program/edit-person-affected-popup/edit-person-affected-popup.component.html

```diff
@@ -129,6 +131,7 @@ <h2>{{ person?.name }}</h2>
                 | translate
             "
             [value]="attributeValues?.phoneNumber"
+            [isRequired]="!program.allowEmptyPhoneNumber"
```

Comment:
if I change OCW to allowEmptyPhoneNumber, than updating this still fails because of the Visa phonenumber requirement. Should we relate this functionality more to each other? E.g. don't allow setting allowEmptyPhoneNumber=true when there is such a FSP requirement? Or good enough for now to let it fail like this. Do we actually still have a use case at all for allowEmptyPhoneNumber after this item?
![image](https://github.com/user-attachments/assets/65b9c6fa-990d-43b9-882c-b5718103e009)

Reply:
Yes, I discussed this with Tys.